repose-aggregator/services/atom-feed/impl/src/main/resources/META-INF/schema/config/atom-feed-service.xsd

```diff
@@ -0,0 +1,189 @@
+           xmlns:html="http://www.w3.org/1999/xhtml"
+        >
+
+    <xs:element name="atom-feed-service" type="AtomFeedServiceConfigType"/>
+
+    <xs:complexType name="AtomFeedServiceConfigType">
+        <xs:sequence>
+            <xs:element name="feed" type="AtomFeedConfigType" minOccurs="1" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="AtomFeedConfigType">
+        <xs:all minOccurs="0">
+            <xs:element name="authentication" type="AuthenticationType">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            Defines the authentication mechanism to be used, and provides any necessary configuration.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:all>
+
+        <xs:attribute name="id" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines a unique ID which may be used to identify this feed.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="uri" type="xs:anyURI" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines the URI at which this feed can be read.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="polling-frequency" type="PositiveIntType" use="optional" default="5">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines how often, in seconds, the Atom Feed service checks this feed for updates.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="entry-order" type="EntryOrderType" use="optional" default="update">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Defines the order in which new Atom events will be delivered to any listeners registered
+                        on this feed.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="EntryOrderType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="random">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            With this ordering, a feed listener will receive events with no ordering guarantees.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+
+            <xs:enumeration value="update">
```

Comment:
This seems like it would be the publish order. where is the order we talked about being the default which was the feed read order?